plugins/importexport/native/NativeImportExportPlugin.inc.php

```diff
@@ -237,21 +237,29 @@ function exportSubmissions($submissionIds, $context, $user, $opts = array()) {
 		assert(count($nativeExportFilters) == 1); // Assert only a single serialization filter
 		$exportFilter = array_shift($nativeExportFilters);
 		$exportFilter->setDeployment(new NativeImportExportDeployment($context, $user));
+
 		$submissions = array();
 		foreach ($submissionIds as $submissionId) {
-			$submission = $submissionDao->getById($submissionId, $context->getId());
+			/** @var $submissionService APP\Services\SubmissionService */
+			$submissionService = Services::get('submission');
```

Comment:
This removes the check to make sure that the submission is in the right context. I'd love to see this re-added upstream (e.g. in `PKPSubmissionService::get`) -- ping @NateWr -- but we probably need to talk about standardizing on that post-3.2. For now, please add a check here by hand, e.g.:
```
if ($submission->getData('contextId') != $context->getId()) continue;
```

Reply:
> I'd love to see this re-added upstream

I'd like to avoid this if we can, because it introduces inconsistencies in the `EntityReadInterface`, but let's chat about this post-3.2.